packages/plugin-ext/src/plugin/documents.ts

```diff
@@ -248,6 +248,13 @@ export class DocumentsExtImpl implements DocumentsExt {
             };
         }
         await this.proxy.$tryOpenDocument(uri, documentOptions);
+
+        if (!options) {
```

Comment:
Maybe it worth mentioning in the method's doc that omitting the `options` parameter actually means opening a document without showing it on UI? To make it a bit clearer...
And adding the link here to the related [issue](https://github.com/theia-ide/theia/issues/5079) might help someone quicker understand why it has been implemented in such way and this is actually a temporary fix.

Reply:
Added comment with dependent issue and [jsdoc](https://github.com/theia-ide/theia/pull/5080/files#diff-83b3c846f689e90d8a3d8755843f4f0fR204)